app/src/main/java/com/nexenio/bleindoorpositioningdemo/ui/beaconview/chart/BeaconChartFragment.java

```diff
@@ -28,10 +30,39 @@
 
     private BeaconChart beaconChart;
 
+    protected BeaconManager beaconManager = BeaconManager.getInstance();
```

Comment:
No need to hold the manager as you can access him anytime by `BeaconManager.getInstance()`.